### ISSUE TITLE
Where are W and Wt in the code?

### ISSUE THREAD

**Issue by patrickfleith** [imperial-qore/TranAD#62]
Hey, I struggle to find out where are `Wt` in the code (the paper refers to as local contextual window of length *K*) and where is `W = {W1, W2, ..., Wt }` in the code denoted as a sequence of sliding windows from input time series T after replication padding.

**Comment by aatifjiwani** [comment]
@patrickfleith The windowing in the code is spread throughout, this is what I have come to understand:
- In the `TranAD*` models in `src/models.py`, the current window `W_t` corresponds to variable `tgt` and `C`, or all points in time up until the current timestamp, corresponds to variable `src`. 
- In `backprop()` in `main.py`, we can see that we're grabbing the current window from the `DataLoader` (line 261)
- The 2D timeseries is converted into a 3D windowed-timeseries, in other terms creating `W = {W1, W2, ..., Wt}`, in `convert_to_windows()` in `main.py`. 

Hope this helps!

**Comment by shreshthtuli** [comment]
Thanks @aatifjiwani for the clarification!
